dispatchers.ts

```diff
@@ -21,6 +22,9 @@ export class CommandDispatcher implements ICommandDispatcher {
 				return;
 			}
 
+			this.$logger.trace("System information:");
+			this.$logger.trace(this.$sysInfo.getSysInfo());
```

Comment:
If the log level is not trace, will the getSysInfo code be executed and just hidden from the output or the method will not be called at all?